confidence_edge_cases/github_context.py

```
@@ -0,0 +1,21 @@
+def read_customer_export(export_root, requested_name):
+    export_path = os.path.join(export_root, requested_name)
+    with open(export_path, "r", encoding="utf-8") as handle:
+        return handle.read()
+
+
+def average_latency(total_latency_ms, request_count):
+    return total_latency_ms / request_count
+
+
+def build_reset_token(email):
+    return hashlib.md5(email.encode("utf-8")).hexdigest()
```

Ruff (0.15.20)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)